package.json

```diff
@@ -19,10 +19,12 @@
     "@emotion/styled": "^11.6.0",
     "@mantine/core": "^6.0.13",
     "@mantine/hooks": "^6.0.13",
+    "@paoliniluis/react-ace-min": "^10.1.11",
```

Comment:
I'm not a fan of having forked version as a part of repo. At least it could be created under metabase org

Reply:
Agree, not a fan of forks as well. The issue here is that react-ace is completely unmaintained and ships with lots of trash, that's why I went with a fork

Reply:
> At least it could be created under metabase org

I agree, and I think it's necessary.

Reply:
> upgrade react-ace (made a fork of the original package, trimmed some useless functionality and upgraded ace to the latest version)

These changes ("trimmed some useless functionality") should also be reviewed by our team.

Reply:
agree @kamilmielnik, if you see the original package it will pull a plugin we don't need (diff.js, split.js) 
![image](https://github.com/metabase/metabase/assets/1711649/3c4f725d-f6de-4614-b054-67ffdd4720de). I wiped those